core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -181,6 +181,7 @@ type DbEthTx struct {
 	// InitialBroadcastAt is recorded once, the first ever time this eth_tx is sent
```

Comment:
Good call! I was trying to avoid touching it in this PR to avoid scope creep but the changes seemed minimal enough. I made the updates in the latest commit